### PR TITLE
Naga holding icons

### PR DESCRIPTION
**Pull request by sejtalk**
## Changelog:
- Added Naga holding icons.

## Developer changelog:
- Added oldgodsgfx that used norse shields.

# How to test:

### CODE REVIEW

**Comment by zumbak04** [comment]
What are this and that things?
<details>
<summary>Click to expand</summary>

image

</details>

File: common/religions/void_group.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@ void_group = {
 	}
 	old_gods_worship = {
 		has_coa_on_barony_only = no
-		graphical_culture = norsegfx
+		graphical_culture = oldgodsgfx
```
Comment (zumbak04) [inline common/religions/void_group.txt:19]: Map shields changed from Norse shields to Christian ones. You should assign Norse shields to this new graphical culture.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religions/void_group.txt (hunk 1 above)
Comment (sejtalk) [reply common/religions/void_group.txt:19]: how?

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**